.circleci/config.yml

```diff
@@ -148,15 +148,15 @@ jobs:
             fi
   run_rainforest:
     docker:
-      - image: rainforestapp/rainforest-cli:latest
+      - image: gcr.io/rf-public-images/rainforest-cli:latest
     steps:
       - run:
           name: Run Rainforest
           command:
-            rainforest-cli run --token "$RAINFOREST_TOKEN" --environment-id 356 --folder 3982 --conflict abort --description "tester-chrome-extension - $CIRCLE_BRANCH $CIRCLE_BUILD_NUM"
+            rainforest-cli run --token "$RAINFOREST_TOKEN" --environment-id 356 --run-group 6940 --crowd automation --conflict abort --description "tester-chrome-extension - $CIRCLE_BRANCH $CIRCLE_BUILD_NUM"
```

Comment:
The run group is the "extension smoke suite" (same as https://github.com/rainforestapp/schrute/blob/master/.circleci/config.yml#L95)